src/lightning/data/streaming/reader.py

```diff
@@ -135,11 +133,6 @@ def run(self) -> None:
             if self._max_cache_size:
                 self._maybe_delete_chunks()
 
-            if _get_from_queue(self._to_stop_queue):
-                return
-
-            sleep(0.05)
```

Comment:
If none of the above if-statement conditions are true, then this while loop will consume a lot of cycles. Shouldn't we keep the sleep?

Reply:
This isn't required now as the reader would wait on the delete queue for up to 5 seconds.